### DISCUSSION TITLE
新的一年奖品也将同步更新，大家想要什么奖品可以在这里留言？

### DISCUSSION THREAD

**Discussion by itcodes** [xitu/gold-miner#8039] [General]
不限于之前的奖品，也可以提一些新的奖品出来？

**Comment by PassionPenguin** [comment]
手办哈哈哈哈哈哈

**Comment by Chorer** [comment]
掘金定制纪念帽？感觉蛮好看 + 实用的，但是礼物列表里没看到
![image](https://user-images.githubusercontent.com/43442630/108586959-437fd980-738c-11eb-9d8e-8450c19d5481.png)

**Reply by itcodes** [reply]
更新

**Reply by Chorer** [reply]
> 更新

没看到有这个礼品。而且发微信联系你也没有回复。

**Comment by Gesj-yeah** [comment]
抱枕 :tada:

**Reply by itcodes** [reply]
这个需要做一下。我记录一下。

**Comment by PassionPenguin** [comment]
https://shop.spreadshirt.com/mdn-store/mdn+15th+anniversary+positive-A5f15e39d1cbf3a0e1303ee21?productType=268&sellable=oroZma9k29sX3vD055mz-268-32&appearance=544&size=29

**Comment by KimYangOfCat** [comment]
@itcodes 在[积分礼物更新（3.11）](https://github.com/xitu/gold-miner/discussions/8198)中，K8机械键盘的产品图似乎选错了？选成了和 K2 一样的，我在网上看到的 K8 是这样的：
![image](https://user-images.githubusercontent.com/32960305/130833209-e34036e7-fd0d-4fe2-9473-de7ae624c521.png)

**Reply by jaredliw** [reply]
这里顺带问一下，K2 和 K8 的轴体有哪些呀？

**Comment by PassionPenguin** [comment]
iPad air 4……

应该是个不错的入手机会

**Reply by PassionPenguin** [reply]
@itcodes 赶紧上线处理

**Comment by PassionPenguin** [comment]
@itcodes @husiyu 上线处理礼品兑换。

**Comment by PassionPenguin** [comment]
赶紧上线处理礼品兑换。

**Reply by PassionPenguin** [reply]
赶紧上线处理礼品兑换。
